src/handlebars/installation.handlebars

```diff
@@ -2,57 +2,72 @@
 
 <main class="grey-bg">
 
-  <h1 class="large-title">Installation</h1>
-
-  <h2><a href="./migration.html" target="_blank">Migration Guide</a></h2>
-
-  <div class="align-center">
-    <div class="btn-container">
-      <form id="jdk-selector" class="btn-form">
-        <h3>1. Choose a Version</h3>
-      </form>
-      <form id="jvm-selector" class="btn-form">
-        <h3>2. Choose a JVM</h3>
-      </form>
+  <h1 class="margin-bottom large-title">Installation</h1>
+
+  <div class="align-left support">
+    <div class="margin-bottom">
+      <p>AdoptOpenJDK binaries are available for download in the following types of installation package:
```

Comment:
I think the HTML here is not quite correct?  open `p` tag with a `ul` list as a child and then another `p` child...

Reply:
Yup - fixing.